clients/lcdproc/disk.h

```diff
@@ -1,6 +1,6 @@
 #ifndef DISK_H
 #define DISK_H
 
```

Comment:
Any reasons you did not include elektragen.h everywhere where Elektra is needed?

(Otherwise it might lead to a compile error if header files are reordered.)

Reply:
You are right, it should be included here.